packages/shell-api/package.json

```diff
@@ -52,10 +52,10 @@
     "@mongodb-js/prettier-config-devtools": "^1.0.1",
     "@mongodb-js/tsconfig-mongosh": "^1.0.0",
     "@mongosh/types": "0.0.0-dev.0",
-    "bson": "^6.8.0",
+    "bson": "^6.9.0",
     "depcheck": "^1.4.7",
     "eslint": "^7.25.0",
-    "mongodb": "^6.9.0",
+    "mongodb": "^6.10.0",
```

Comment:
Yeah, for the bulk writes, we'll need to https://jira.mongodb.org/browse/MONGOSH-1100 ... for `maxTimeMS`, we'll still need to do a bit of work in Compass, because the Node.js driver team didn't go with an approach that would make our current code "just work"